### DISCUSSION TITLE
Frozen solving environment step in installing moose in linux

### DISCUSSION THREAD

**Discussion by Ali1990dashti** [idaholab/moose#18093] [Q&A Getting Started] [answered]
Dear community,
Since yesterday I have bee tryin to install moose. I am usnig `ubunu 20.04` and `conda 4.8.3.` I exactly followed [explained instructions](https://mooseframework.inl.gov/getting_started/installation/conda.html). But, when I try the line `conda create --name moose moose-libmesh moose-tools`  it stucks in the solving environment step. I tried to install it also in my base environemnt and also out of conda but the results was the same and I stuck in the same step. Uploaded fig also shows it I do appreciate any help to solve my issue.
![moose_install](https://user-images.githubusercontent.com/62764899/122186047-eacdd280-ce8d-11eb-9046-8a7b3e06c1b5.jpg)

**Comment by GiudGiud** [comment, marked as answer]
Try
 conda create -n moose -q -y
 conda activate moose
 conda install moose-tools
 conda install moose-libmesh
 
 It solves faster

**Reply by Ali1990dashti** [reply]
Dear @GiudGiud ,

Thanks a lot.
Your solution fixed my problem.
